api/app/Console/Commands/SyncNewRodFields.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Enums\CandidateRemovalReason;
+use App\Enums\PoolCandidateStatus;
+use App\Models\PoolCandidate;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class SyncNewRodFields extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:sync-rod-fields';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Where the new RoD fields should be non-null, insert an appropriate value';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $dateNow = Carbon::now();
+        $placedStatuses = PoolCandidateStatus::placedGroup();
+        $removedStatuses = PoolCandidateStatus::removedGroup();
+        $finalDecisionStatuses = [
+            PoolCandidateStatus::QUALIFIED_AVAILABLE->name,
+            PoolCandidateStatus::QUALIFIED_UNAVAILABLE->name,
+            PoolCandidateStatus::QUALIFIED_WITHDREW->name,
+            PoolCandidateStatus::PLACED_CASUAL->name,
+            PoolCandidateStatus::PLACED_TERM->name,
+            PoolCandidateStatus::PLACED_INDETERMINATE->name,
+            PoolCandidateStatus::EXPIRED->name,
+            PoolCandidateStatus::SCREENED_OUT_APPLICATION->name,
+            PoolCandidateStatus::SCREENED_OUT_ASSESSMENT->name,
+        ];
+
+        $applicableModels = PoolCandidate::with('user')
```

Comment:
Lazy load error if not present, possible to do with selectable columns and text search on the user model

Reply:
I feel like your comments here should be comments in the actual code! Instead of just helping me review it now, help anyone reading this code in the future.

Reply:
Yeah, that one probably should have been in there hehe https://github.com/GCTC-NTGC/gc-digital-talent/pull/10150/commits/3e942409eec5482dd06aaccb57ec4dd1488b87c2